lib/management/group.ts

```diff
@@ -0,0 +1,33 @@
+import { SdkResponse, transformResponse } from '@descope/core-js-sdk';
+import { CoreSdk } from '../types';
+import apiPaths from './paths';
+import { Group } from './types';
+
+const withGroup = (sdk: CoreSdk, managementKey?: string) => ({
```

Comment:
It would be nice to have function documentation. I know it's missing almost everywhere, but I started adding it (see the `user.loadByJwtSubject` function.

Reply:
@itaihanski done